service-base/src/test/java/org/eclipse/hono/service/http/DefaultFailureHandlerTest.java

```diff
@@ -13,21 +13,28 @@
 
 package org.eclipse.hono.service.http;
 
-import static org.hamcrest.CoreMatchers.is;
-import static org.junit.Assert.*;
-import static org.mockito.Mockito.*;
-
-import java.net.HttpURLConnection;
-
+import io.netty.handler.codec.http.HttpResponseStatus;
+import io.vertx.core.buffer.Buffer;
+import io.vertx.core.http.HttpServerRequest;
+import io.vertx.core.http.HttpServerResponse;
+import io.vertx.core.http.impl.HttpServerResponseImpl;
+import io.vertx.ext.web.RoutingContext;
 import org.eclipse.hono.client.ClientErrorException;
 import org.junit.Test;
 import org.mockito.ArgumentCaptor;
 import org.mockito.Mockito;
 
-import io.vertx.core.buffer.Buffer;
-import io.vertx.core.http.HttpServerRequest;
-import io.vertx.core.http.HttpServerResponse;
-import io.vertx.ext.web.RoutingContext;
+import java.net.HttpURLConnection;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertThat;
+import static org.mockito.Mockito.RETURNS_MOCKS;
```

Comment:
please remove this static import

Reply:
Below static imports already exists. As part of this PR, I replaced  import `import <package>.*` as to exact classes to be used. Also I noticed that in our test classes often static imports are being used. 

```
import static org.hamcrest.CoreMatchers.is;
import static org.junit.Assert.*;
import static org.mockito.Mockito.*;
```

Reply:
We always refer to constants defined in other classes using the defining class name and the constant name. You are doing it yourself in the code:

```
final HttpServerRequest request = mock(HttpServerRequest.class, Mockito.RETURNS_MOCKS);
final HttpServerResponse response = mock(HttpServerResponseImpl.class, RETURNS_MOCKS);
```

Reply:
Agreed, I will append the class name too.